scripts/deployment/l2/globals_ethereum_mainnet.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
wrong here

Reply:
let's keep it here for now

Reply:
I'll merge it back to the main one later